homeassistant/components/igloohome/quality_scale.yaml

```diff
@@ -0,0 +1,68 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: |
+      This integration does not provide additional actions.
+  docs-high-level-description: done
+  docs-installation-instructions: done
```

Comment:
currently you're missing a {% include integrations/config_flow.md %}
and also a {% configuration_basic %} block describing the config parameters

Reply:
@zweckj , are you referring to the documentation for `homeassistant.io`?

Reply:
Done.